src/components/js-tracer/EmptyReasons.js

```diff
@@ -31,9 +31,9 @@ class MarkerChartEmptyReasons extends PureComponent<Props> {
   }
 }
 
-export default explicitConnect<{||}, StateProps, {||}>({
+export const JsTracerEmptyReasons = explicitConnect<{||}, StateProps, {||}>({
```

Comment:
praise: Thanks, this looks like the proper way to export this EmptyReasons component.